source/_meta/_01-foot.twig

```diff
@@ -8,6 +8,40 @@
     <!-- Include all compiled plugins (below), or include individual files as needed -->
     <script src="../../css/bootstrap/js/bootstrap.min.js"></script>
     <script src="../../js/ui-scripts.js"></script>
+    
+    <!-- load CKEditor -->
+    <script>
+        ClassicEditor
```

Comment:
Lets put this inside a jQuery `.ready()` event - so that we can put the JS within the Twig template & still be sure it doesn't run until page DOM is loaded.